runtime/src/lib.rs

```diff
@@ -935,7 +937,7 @@ parameter_types! {
 	/// Not strictly enforced, but used for weight estimation.
 	pub const MaxLocks: u32 = 50;
 	/// TODO: Documentation
-	pub const IsTransferable: bool = false;
+	pub const IsTransferable: bool = true; // TODO Review - Change this back to `false` once we update our substrate dependency
```

Comment:
so you wanna merge this first then fix substrate?

Reply:
yeah actually it makes more sense to fix substrate first. since we don't want `IsTransferable` to ever be `true`

Reply:
it does not matter for `devel` as long as substrate fix comes next though.

Reply:
> it does not matter for `devel` as long as substrate fix comes next though.

okay let's do that then. otherwise, if I merge the substrate PR first - there might be merge conflicts